runtime/Go/antlr/parser.go

```diff
@@ -274,7 +274,7 @@ func (p *BaseParser) RemoveParseListener(listener ParseTreeListener) {
 }
 
 // Remove all parse listeners.
```

Comment:
Documentation should be updated to match go standards:

`// RemoveParseListeners removes all parse listeners.`